services/nostr-band/nostr-band-followers.service.js

```diff
@@ -0,0 +1,57 @@
+import Joi from 'joi'
+import { metric } from '../text-formatters.js'
+import { BaseJsonService } from '../index.js'
+
+const npubSchema = Joi.object({
+  followers_pubkey_count: Joi.number().required(),
+}).required()
+
+const mainSchema = Joi.object({
+  stats: Joi.object().pattern(Joi.string(), npubSchema).required(),
+}).required()
+
+export default class NostrBandFollowers extends BaseJsonService {
+  static category = 'social'
+
+  static route = {
+    base: 'nostr-band/followers',
+    pattern: ':npub',
+  }
+
+  static examples = [
+    {
+      title: 'Nostr.band Followers',
+      namedParams: {
+        npub: 'npub18c556t7n8xa3df2q82rwxejfglw5przds7sqvefylzjh8tjne28qld0we7',
+      },
+      staticPreview: this.render({ followers: 720 }),
+    },
+  ]
+
+  static defaultBadgeData = { label: 'followers', namedLogo: 'nostr' }
```

Comment:
This logo does not exist in the simple-icons set. Can we remove it

Reply:
done: https://github.com/badges/shields/pull/9870/commits/bc6faa834fb85bfa8cc8f186f1a08d010fd2dc87